packages/core/src/index.js

```diff
@@ -28,6 +29,7 @@ export { default as QuadUv } from './utils/QuadUv';
 export { default as checkMaxIfStatementsInShader } from './shader/utils/checkMaxIfStatementsInShader';
 export { default as Shader } from './shader/Shader';
 export { default as Program } from './shader/Program';
+export { default as GLProgram } from './shader/GLProgram';
 export { default as UniformGroup } from './shader/UniformGroup';
 export { default as SpriteMaskFilter } from './filters/spriteMask/SpriteMaskFilter';
 export { default as Filter } from './filters/Filter';
```

Comment:
Wow, I must really be off-base about default exports or something changed recently?